<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RV6F1IE" failures="1" tests="5" name="Shopping Test" time="22.975" errors="0" timestamp="2022-12-10T03:50:40 IST">
  <testcase classname="purushotham.tests.HashMapTest" name="HashMapOrder" time="5.418"/>
  <testcase classname="purushotham.tests.Shopping" name="clientApp" time="5.069"/>
  <testcase classname="purushotham.tests.Shopping" name="verifyTheOrder" time="0.95">
    <skipped/>
  </testcase> <!-- verifyTheOrder -->
  <testcase classname="purushotham.tests.HashMapTest" name="HashMapOrder" time="5.233"/>
  <testcase classname="purushotham.tests.Shopping" name="verifyTheOrder" time="0.699">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at purushotham.tests.Shopping.verifyTheOrder(Shopping.java:37)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyTheOrder -->
</testsuite> <!-- Shopping Test -->
